src/config/hoard.rs

```diff
@@ -137,12 +151,18 @@ impl Pile {
     ///
     /// Various sorts of I/O errors as the different [`enum@Error`] variants.
     pub fn backup(&self, prefix: &Path) -> Result<(), Error> {
-        // TODO: do stuff with config
+        // TODO: do stuff with pile config
         if let Some(path) = &self.path {
-            log::trace!("Backing up {}", path.to_string_lossy());
+            let _span = tracing::debug_span!(
+                "backup_pile",
+                path = path.to_string_lossy().as_ref(),
+                prefix = prefix.to_string_lossy().as_ref()
+            )
+            .entered();
+
             Self::copy(path, prefix)?;
         } else {
-            log::warn!("Pile has no associated path -- perhaps no environment matched");
+            tracing::warn!("pile has no associated path -- perhaps no environment matched?");
```

Comment:
Warning? Info?

Reply:
I think leaving this as a warning is acceptable here. If the user meant to have this pile match the current system, it is good to draw attention to the lack of match. OTOH, making this a warning when the user intentionally doesn't want it to match will likely lead to the user generally ignoring further warnings. OTOOH, this warns once per pile, so there shouldn't be *too* many warnings....

I am going to leave it as-is until it annoys me or another user enough to want it changed.